src/test/resources/emails/sessionLinksRecoveryNoSessionsFoundEmail.html

```diff
@@ -0,0 +1,22 @@
+<p>Hello,</p>
+
+<p>
+    You are receiving this email because a request to resend TEAMMATES links associated with the email address <strong>student1InCourse1@gmail.tmt</strong> was made on <a href="http://localhost:4200/">TEAMMATES</a>.
+</p>
+
+<p>
+    We could not find any sessions associated with this email address that have opened or closed in the past 180 days.</p>
+<p>
+    If a session that you expected to see is missing, it may be because it is associated with a different email address. In that case, you can <a href=http://localhost:4200/web/front/help/session-links-recovery>try again</a> with a different email address or contact your course instructor(s) to check whether this email address is the one used to enroll you in TEAMMATES.
+    Also, sessions that have been not opened or closed in the past 180 days are not shown. If you want the links to those sessions, contact the TEAMMATES support team mentioned below.
+</p>
+
+<p>
+    <strong>Technical help regarding TEAMMATES</strong>:
+    Email TEAMMATES support team at app_admin@gmail.com.
```

Comment:
`app_admin@gmail.com` should not be hardcoded here. Did you use snapshot testing (update mode) to generate this and other similar emails?